src/utils/CourseChecklist/courseChecklistData.jsx

```diff
@@ -5,72 +5,50 @@ export const filters = {
 };
 
 export const launchChecklist = {
```

Comment:
I'm not sure how useful this file is anymore. I could just move all this data into the component itself. Thoughts?

Reply:
Which component were you thinking of moving it into? Do you think it would be easier to understand if it were to be moved there?